src/utils/isOutsideClick.js

```diff
@@ -12,8 +12,7 @@ const getPath = e => {
   return path
 }
 
-export default (e, className) => {
+export default (e, node) => {
   if (!(e instanceof Event)) return false
-  const completeClassName = className ? `${className} react-dropdown-tree-select` : 'react-dropdown-tree-select'
-  return !getPath(e).some(node => node.className && node.className.indexOf(completeClassName) >= 0)
+  return !getPath(e).some(eventNode => eventNode === node)
```

Comment:
This is what `node.contains` was supposed to do but the native method was not reliable across browsers. We had issues in firefox which is why `isOutsideClick` was born. Hopefully, this implementation will fare better.

Reply:
Did you check on the node itself also? Or what was the problem? Just to ensure it's not introduced again. I've only tested it with latest ff, chrome, edge and ie11.

Reply:
This is one of the early attempts that caused issues on Firefox: 

https://github.com/dowjones/react-dropdown-tree-select/blob/bbd5c2ca62cb4f563dc49c8a7bcf680fa872370f/src/index.js#L89-L95

Reply:
Ok not sure why that would not work. Any steps to reproduce the issue to verify it's not reintroduced?

Reply:
If it works on the latest FF, then I think it's a good enough indicator that it fixes the issue. Let's roll with that for now.